src/mds/MDSMap.h

```diff
@@ -237,12 +242,15 @@ class MDSMap {
     return utime_t(session_timeout,0);
   }
   uint64_t get_max_filesize() { return max_file_size; }
+  void set_max_filesize(uint64_t m) { max_file_size = m; }
   
   int get_flags() const { return flags; }
   int test_flag(int f) const { return flags & f; }
   void set_flag(int f) { flags |= f; }
   void clear_flag(int f) { flags &= ~f; }
 
+  const std::string &get_fs_name() const {return fs_name;}
```

Comment:
I'm not sure I like the mixed use of fs_name and ns to identify a file system.  Especially with the rados namespace in teh file_layout_t info now too (although that field is named pool_ns, and the user-facing virtual xattrs are pool_namespace).